framework/bug-mining/promote-to-db.pl

```diff
@@ -260,6 +266,17 @@ sub _cp {
     }
 }
 
+sub _ap {
+    my ($src, $dst) = @_;
+    print "\t... appending $src -> $dst\n";
+    $dst =~ m[^(.*)/.*$];
+    system ("mkdir -p $1") == 0 or die "could not mkdir dest $1: $!";
+    if (-e $src) {
+        system("cat $src >> $dst") == 0 or die "could not append $src: $!";
+        print "\t... OK\n";
+    }
+}
+
 sub _db_cp {
```

Comment:
Should we change this name as well (i.e., make it more expressive)?

Reply:
I will make a pass on all those abbreviations.